### PR TITLE
Integrated coreUI

### PR DESCRIPTION
**Pull request by vpn1997**
* Integrated coreUI with Origami.
 * All coreUI packages in ui folder in src.
 * Description - In this Pr I have integrated coreUI with Origami.So origami now has a modified layout and a working homepage.I'll raise the PRs for remaining pages subsequently .

### CODE REVIEW

**Comment by AvaisP** [comment]
@vpn1997 Looks good. For the rest of the UI PRs, upload a working video too if possible :smile:

File: Origami/src/components/nav_root.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  { path: '/', exact: true, name: 'Home', component: DefaultLayout },
+  { path: '/u/:username/:user_id', name: 'user', component: ShareProfile },
+  { path: '/p/:shorturl', exact: true, name: 'short', component: URLShortenerComponent },
+  { path: '/demo_register', exact: true, name: 'Create demo', component: RegisterNonGHPageComponent },
+  { path: '/login*', name: 'login', component: LoginComponent },
+  { path: '/ngh/user',exact: true,name:'My Demo', component:NonGHUserProfileComponent },
+
+];
+
+export default routes;
```
Comment (AvaisP) [inline Origami/src/components/nav_root.js]: Add new line

File: Origami/src/components/routes.js [hunk 2]
```diff
@@ -16,15 +16,26 @@ import SelectInputComponentsNonGHPageComponent from "./deployment/UsePrebuiltPro
...
+    <Route  path="/" component={DefaultLayout} />
+    <Route exact path="/home" component={HomePageComponent} />
+    <Route exact path="/profile" component={LoginComponent} />
     <Route exact path="/ngh/user" component={NonGHUserProfileComponent} />
+
+
     <Route
       exact
-      path="/ngh/user/register"
+      path="/demo_register"
```
Comment (AvaisP) [inline Origami/src/components/routes.js]: Why?

Comment (vpn1997) [reply Origami/src/components/routes.js]: Hey @AvaisP  i didnt understand the use or implication of ngh and i guess its a legacy code(?).Thats why i modified the url.

File: Origami/src/components/nav_root.js (hunk 1 above)
Comment (vpn1997) [reply Origami/src/components/nav_root.js]: done

File: Origami/src/components/routes.js (hunk 2 above)
Comment (AvaisP) [reply Origami/src/components/routes.js]: Cool

**Pull request merged**